recommendations/index.md

```diff
@@ -9,13 +9,14 @@ are beyond the scope of the base JSON API specification.
 
 ## Recommendations for Naming <a href="#naming" id="naming" class="headerlink"></a>
 
-It is recommended that resource types, attribute names, and association
-names be "dasherized"; i.e. consist of only lower case alphanumeric
-characters and dashes (U+002D HYPHEN-MINUS, "-").
-
-It is also recommended that resource types be pluralized. Dasherized and
-pluralized resource types can be used as URL segments without translation,
-as discussed below.
+The allowed and recommended characters for an URL safe naming of members are defined in the format spec. In addition, to standardize member names, the following restrictions are recommended:
+- Member names **SHOULD** be all lower-case.
+- Member names **SHOULD NOT** start with a digit (U+0030 to U+0039, "0-9")
+- Member names cointaining multiple words **SHOULD** use a hyphen minus (U+002D HYPHEN-MINUS, "-") as word separator.
+
+Together with the recommendations in the format spec this results in:
+- Member names **SHOULD** start and end with the characters "a-z" (U+0061 to U+007A)
+- Member names **SHOULD** contain only the characters "a-z" (U+0061 to U+007A), "0-9" (U+0030 to U+0039) and "-" (U+002D HYPHEN-MINUS)
```

Comment:
Why not just discard all that is in this PR and specify only those two rules, then? It seems simpler and much clearer (the long lists in the spec especially are really tedious).